doc_src/gear_developers/limitations.md

```diff
@@ -0,0 +1,12 @@
+# Limitations
+
+There are certain limitations in Antikythera.
+We introduce some major limitations here. But this does not cover all the limitations of Antikythera. Please also read other parts of the documentation when you implement gear.
+
+* Execution time of web/g2g request
+  * Please refer [here](https://hexdocs.pm/antikythera/development_environment.html#environment-variables-to-tweak-behavior-of-antikythera).
```

Comment:
I felt difficult to find further information in the referenced page.

How about the following?

```markdown
- Execution time of web/g2g request
    - Set via `GEAR_ACTION_TIMEOUT`. Please refer [here](https://hexdocs.pm/antikythera/development_environment.html#environment-variables-to-tweak-behavior-of-antikythera).
- Execution time of AsyncJob
    - Set via `max_duration`. Please refer [here](https://hexdocs.pm/antikythera/Antikythera.AsyncJob.html#module-registering-jobs).
    - AsyncJob has other limitations too.
- Heap size which a process can use
    - Set via `GEAR_PROCESS_MAX_HEAP_SIZE`. Please refer [here](https://hexdocs.pm/antikythera/development_environment.html#environment-variables-to-tweak-behavior-of-antikythera).
```

Reply:
I think it is better.
I fixed in 7bd85ef75f90e21a65472fbd64b07ee6921ad484